### PR TITLE
Use RPM-GPG-KEY-foreman from foreman-release

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
@ehelms not sure I follow your comment. My mumbling that this should live in `foreman-release` (in https://github.com/theforeman/tool_belt/pull/205/) was still suggesting a separate sub-package, that would not be published to yum.tfm.o/releases.

File: packages/katello/katello-repos/katello-repos.spec [hunk 1]
```diff
@@ -17,11 +17,12 @@ Group:          Applications/Internet
...
 URL:            https://theforeman.org/plugins/katello/
 Source0:        katello.repo
-Source1:        RPM-GPG-KEY-katello-2015
 
 BuildArch:      noarch
 
-BuildRequires: sed
+BuildRequires:  sed
+
+Requires:       foreman-release
```
Comment (ekohl) [inline packages/katello/katello-repos/katello-repos.spec]: Do we want to enforce a match here?
```suggestion
Requires:       foreman-release >= 1.24
Requires:       foreman-release < 1.25
```

**Comment by ekohl** [comment]
My idea was a `katello` subpackage in `foreman-release`. I'd also publish that to yum.tf.o in the main foreman repo. That way the katello installation instructions become:

```
yum -y install epel-release http://yum.puppet.com/puppet-release-el-7.noarch.rpm https://yum.theforeman.org/releases/latest/el7/x86_64/foreman-release.rpm
yum -y install foreman-release-katello
yum -y install foreman-installer-katello
foreman-installer --scenario katello
```

**Comment by ehelms** [comment]
What @ekohl said here https://github.com/theforeman/foreman-packaging/pull/4208#issuecomment-544525306 much more elegantly than I did :P

File: packages/katello/katello-repos/katello-repos.spec (hunk 1 above)
Comment (evgeni) [reply packages/katello/katello-repos/katello-repos.spec]: That would require updating the lock on every major release. I'd defer that until we merge the two packages? :)

Comment (ekohl) [reply packages/katello/katello-repos/katello-repos.spec]: I guess that's fine. Users will find out it's incompatible anyway when the GPG signatures don't match ;)

**Comment by evgeni** [comment]
Oh, you wrote "advantageous" not "adventurous", that changes everything.

I think moving this completely into `f-release` is the right thing to do, but requires more work, while this PR get's us most of the benefits (don't have to care about `k-repos` besides version-bump) without the headache of the move. So I'd prefer to merge this and drop "merge `f-release` and `k-repos`" to the backlog?

**Pull request merged**